includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -233,23 +233,35 @@ public function getAllElements() {
 		$params = $this->getRequestParameters();
 
 		$this->setResponseData( [
-			'global-footer' => ( new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] ) )->getData(),
+			'global-footer' => ( new DesignSystemGlobalFooterModel( $params[ 'product' ], $params[ 'id' ], $params[ 'lang' ] ) )->getData(),
 			'global-navigation' => $this->data
 		] );
 
 		$this->addCachingHeaders();
 	}
 
 	private function getRequestParameters() {
-		$wikiId = $this->getRequiredParam( 'wikiId' );
+		// ultimately, id will be a required param, but fall back to wikiId while transitioning
```

Comment:
This is trickier than https://github.com/Wikia/app/pull/11245/files#diff-4a1b823158dfc4984cc02216f2910d32R250 to synchronize - it looks more cost-efficient to do it like you're proposing. Can you link a ticket to cleanup in those comments?